recipes/pydantic-mongo/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pydantic-mongo" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pydantic_mongo-{{ version }}.tar.gz
+  sha256: d1b05d6bf28e0eeca66809a99c362b958bf6fc6af59350d44d9a948e5fc30e3b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=42
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - pymongo >=4.3,<5.0
+    - pydantic >=2.0.2,<3.0.0
+
+test:
+  source_files:
+    - test/
+  imports:
+    - pydantic_mongo
+  commands:
+    - pip check
+    - pytest ./test -k 'not test_enhance_meta'  # [not win]
```

Comment:
Just curios but if this is noarch, what failed on Windows here?

Reply:
See [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=964926&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=7c0f8eae-6d6f-51bf-636f-73a1a7fb1bc4):

```
============================= test session starts =============================
platform win32 -- Python 3.12.4, pytest-8.2.2, pluggy-1.5.0
rootdir: %SRC_DIR%
collected 0 items

============================ no tests ran in 0.03s ============================
ERROR: file or directory not found: test_enhance_meta'
```

I think there is something wrong with the `-k` parsing

Reply:
Ah! You may need to sue double quotes. Not a big deal for noarch packages but if it was an arch related failures, then we would need to do something about it.